bedrock/settings/__init__.py

```diff
@@ -179,10 +104,136 @@
     DEV_LANGUAGES = PROD_LANGUAGES
     LANGUAGE_CODE = "en"  # Pocket uses `en` not `en-US`
 
+    # CSP settings for POCKET, expanded upon later:
+    _csp_default_src = [
+        "'self'",
+        "*.getpocket.com",
+    ]
+    _csp_img_src = [
+        "data:",
+        "www.googletagmanager.com",
```

Comment:
Does Pocket use GTM?

I see calls to `www.google-analytics.com` when viewing https://getpocket.com/add, but not `www.googletagmanager.com"`.

Reply:
It's possible we don't need it - we could zap it now, or wait and zap it once we're sure it's not used in production. It's a domain that we trust for Mozorg, at least.

Reply:
Up to you here if / when we want to remove stuff, but from my perspective I think it's good to keep the lists lean and separate so we can really restrict each site to only what's needed.

Reply:
If you want we can always file a "clean up the dual-site CSP mish-mash" issue for after merge? (using a more sensible name of course 😅)

Reply:
Zapping now - we can add back in if pre-launch testing shows we should